documentation/contributing/future_projects/index.markdown

```diff
@@ -17,23 +17,23 @@ If you would like to contribute to these efforts, have a look at the related Git
 
 Feel free to contact [PickNik Robotics](https://picknik.ai/connect/) for further information or sponsorship opportunities.
 
-**Note:** If you are looking at this list for a Google Summer of Code application, you should become familiar with the issue and codebase before starting the project in order to succeed within the new 10-week time frame. Do not
+**Note:** If you are looking at this list for a Google Summer of Code application, you should become familiar with the issue and codebase before starting the project in order to succeed within the new 10-week time frame.
 
 # Table of Contents
-- [Table of Contents](#table-of-contents)
-  - [TrajOpt Integration and Related Work on Trajectory Optimization Methods](#trajopt-integration-and-related-work-on-trajectory-optimization-methods)
-  - [Simultaneous Trajectory Execution](#simultaneous-trajectory-execution)
-  - [Tutorial for multiple robot arms](#tutorial-for-multiple-robot-arms)
-  - [Scene graph support](#scene-graph-support)
-  - [Creation of a Benchmark Suite for Evaluating MoveIt Performance](#creation-of-a-benchmark-suite-for-evaluating-moveit-performance)
-  - [MoveIt 2 Tutorials](#moveit-2-tutorials)
-  - [Add Support for MoveIt Task Constructor to MoveIt 2](#add-support-for-moveit-task-constructor-to-moveit-2)
-  - [Improved Integration with ROS-Controls and Controller Switching](#improved-integration-with-ros-controls-and-controller-switching)
-  - [Improve MoveIt Grasps Library](#improve-moveit-grasps-library)
-  - [Cartesian Planning improvements](#cartesian-planning-improvements)
-  - [Improve Warehouse Support](#improve-warehouse-support)
-  - [MoveIt-OMPL Planning Interface](#moveit-ompl-planning-interface)
-  - [Mobile Base Integration](#mobile-base-integration)
+
+- [TrajOpt Integration and Related Work on Trajectory Optimization Methods](#trajopt-integration-and-related-work-on-trajectory-optimization-methods)
```

Comment:
I auto-generated the TOC with [this VSCode plugin](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one). How about leaving the list formatted as is and removing the `# Table of Contents` header instead?

Reply:
I used the `<!-- omit in toc -->` to clean up the TOC.